bucket/steam-art-manager.json

```
@@ -29,6 +29,6 @@
         "github": "https://github.com/Tormak9970/Steam-Art-Manager"
     },
     "autoupdate": {
-        "url": "https://github.com/Tormak9970/Steam-Art-Manager/releases/download/v3.10.5/steam-art-manager_$version.msi.zip"
+        "url": "https://github.com/Tormak9970/Steam-Art-Manager/releases/download/v$version/steam-art-manager_$version.msi.zip"
```

GitHub tag already includes `v` prefix, so `$version` should be used without a `v` prefix (or `v$cleanVersion` but that's redundant):

```suggestion
        "url": "https://github.com/Tormak9970/Steam-Art-Manager/releases/download/$version/steam-art-manager_$version.msi.zip"
```